CMakeLists.txt

```diff
@@ -33,6 +33,22 @@ option(SLEEF_DISABLE_SSL "Disable testing with the SSL library" OFF)
 option(SLEEF_ENABLE_CUDA "Enable CUDA" OFF)
 option(SLEEF_ENABLE_CXX "Enable C++" OFF)
 
+#
+
+if (DEFINED SLEEF_BUILD_SHARED_LIBS)
+  set(BUILD_SHARED_LIBS ${SLEEF_BUILD_SHARED_LIBS})
+endif ()
+
+if (SLEEF_SHOW_CONFIG)
+  # Normalize the value of BUILD_SHARED_LIBS so that it displays nicely
+  # in the configuration display
+  if (BUILD_SHARED_LIBS)
+    set(BUILD_SHARED_LIBS YES)
+  else ()
+    set(BUILD_SHARED_LIBS NO)
```

Comment:
nit: We tend to use ON/OFF in SLEEF. Would you mind changing to that for consistency? Or is there a point using YES/NO over ON/OFF?

Reply:
Happy to switch to ON/OFF. There's no technical reason to prefer one over the other. I think something about the grammar in the summary led me to YES/NO.

CMake booleans are cursed 😝

Reply:
@blapie - done!